app/src/main/res/layout/fragment_marketplace.xml

```diff
@@ -69,38 +62,85 @@
         </androidx.constraintlayout.widget.ConstraintLayout>
 
         <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/main_constraint"
             android:layout_width="match_parent"
             android:layout_height="0dp"
             android:background="@color/very_light_pink"
             app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/bar_constraint">
 
             <androidx.recyclerview.widget.RecyclerView
                 android:id="@+id/tabs_recycler"
                 android:layout_width="match_parent"
-                android:layout_height="30dp"
-                android:layout_marginStart="22dp"
+                android:layout_height="50dp"
```

Comment:
also here -> why are you using app:layout_constraintStart_toStartOf and app:layout_constraintEnd_toEndOf if you use android:layout_width="match_parent" not 0dp ?

Reply:
👍